charts/portworx-essentials/v1.0.0/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+name: portworx-essentials
+appVersion: 1.0.0
+version: 1.0.0
+description: A Helm chart for installing Portworx Essentials on Kubernetes.
+keywords:
+  - Storage
+  - ICP
+  - persistent disk
+  - pvc
+  - cloud native storage
+  - persistent storage
+  - portworx
+  - amd64
+  - portworx essentials
+  - free
+home: https://portworx.com/
+maintainers:
+  - name: sharma-tapas
+    email: tapas@portworx.com
+sources:
+  - https://github.com/portworx/helm
+icon: https://raw.githubusercontent.com/portworx/px-logos/master/Portworx_Product_Logos/Portworx_Essentials/essentials_full_color_logo/png/portworx-essentials-horizontal-color-500px.png
```

Comment:
Please keep a copy of the logo local in the `portworx-essentials` parent folder and reference it here as `file://../logo.png`. This will allow users running Rancher in an airtight environment to see it.

Reply:
Added the local copy of the logo.